clients/shisui/hive.yaml

```diff
@@ -0,0 +1,2 @@
+roles:
+  - portal
```

Comment:
add a new line to the end of this file

Reply:
This file still indicting an invalid page ending

Reply:
![image](https://github.com/ethereum/hive/assets/31669092/eeac8ee3-7736-4876-b7d8-69e19682cfb2)
by that I mean this red circle. Once it is gone I am good to merge this